machine-learning-notebooks/transfer-learning-custom-azureml/1.Collect_Image_Data_from_PerceptDK.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
I don't know how to leave a comment in a reasonable spot in a python notebook in GitHub reviews. But at some point in the notebook you refer to water bottles, but I think you are actually using office supplies as the example. I could be wrong.

Reply:
Thanks for spotting.  Done.